app/js/components/manageApps/ManageApps.jsx

```diff
@@ -278,8 +278,10 @@ export default class ManageApps extends React.Component {
           uploadStatus: 0,
           showProgress: false,
           files: null,
+          updatesAvailable: null,
```

Comment:
Could you explain how changes in this file are related to the test that the ticket requires?